integration_tests/lock_test.go

```diff
@@ -442,17 +442,17 @@ func (s *testLockSuite) mustGetLock(key []byte) *txnkv.Lock {
 }
 
 func (s *testLockSuite) ttlEquals(x, y uint64) {
-	// NOTE: On ppc64le, all integers are by default unsigned integers,
-	// hence we have to separately cast the value returned by "math.Abs()" function for ppc64le.
-	if runtime.GOARCH == "ppc64le" {
-		s.LessOrEqual(int(-math.Abs(float64(x-y))), 2)
-	} else {
-		s.LessOrEqual(int(math.Abs(float64(x-y))), 2)
+	if x < y {
+		x, y = y, x
 	}
-
+	s.LessOrEqual(x-y, uint64(5))
 }
 
 func (s *testLockSuite) TestLockTTL() {
+	managedLockTTL := atomic.LoadUint64(&tikv.ManagedLockTTL)
+	atomic.StoreUint64(&tikv.ManagedLockTTL, 20000)                // set to 20s
+	defer atomic.StoreUint64(&tikv.ManagedLockTTL, managedLockTTL) // restore value
```

Comment:
`ManagedLockTTL` will be set to 3000 in test. The calculated TTL which is great than `ManagedLockTTL` will be cut to `ManagedLockTTL`, this strategy fails the test.